src/Tank/TankHealthBar.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <SFML/Graphics/RenderWindow.hpp>
+#include <SFML/Graphics/Text.hpp>
+#include <SFML/System/Vector2.hpp>
+
+class TankHealthBar {
+ public:
+  TankHealthBar(unsigned int total_health);
```

Comment:
Let's use "explicit" here to avoid implicit conversions between int and TankHealthBar

Reply:
Fixed, thank you for the review! :)